tempesta_fw/t/functional/run_tests.py

```diff
@@ -79,7 +79,11 @@ def usage():
 
 #run tests
 loader = unittest.TestLoader()
-tests = loader.discover('.')
+if not remainder:
+    tests = loader.discover('.')
```

Comment:
Not a problem of this patchset, just remark on current problems. `discover()` sorts tests in very strange way, e.g. selftests for the test suit is done somewhere in the middle of test cases.